### ISSUE TITLE
Reading order not taken into account ?

### ISSUE THREAD

**Issue by Lexstok** [mtgto/Fomalhaut2#11]
With the latest update, it seems the reading order is not taken into account ? LEFT to RIGHT or RIGHT to LEFT, the comics I've tried opening always force me to read from R to L.

Tested JAN 2024

**Comment by mtgto** [comment]
@Lexstok Thank you for your question.

I also tried but could not reproduce it. I have a few questions to inquiry.

1. There is a "Default Page Order" setting in Settings menu item. This determines the page order of a book that is opened for the first time. However, once a book is opened, the page order is recorded in the DB, so if I change the setting after the book has been opened, it does not take effect. Is this your problem?
2. Are you talking about the page order when displayed in the app? Or are you referring to the page order when viewed in a web browser?
3. In the app, when you open a book, you can change the page order from the toolbar. Is the page order in the toolbar always from right to left on your screen?

**Comment by Lexstok** [comment]
Hello,

I’ve tested again, here are my responses :

I’m using the app.

After the update, everything I newly opened was from R to L. I then changed the default page order via the settings, but it did not take effect. I thought this was the only way to change the page order.

With your explanation, I now understand better that the default page order is ‘setting the default for any new comics that I open", not for changing for existing comics.

Why did I think this ? Because I did not see the icons to change the reading order in the icon bar - my bad, I completely overlooked them !

Using the icons to change the reading order did solve the problem !

So to summarize : 
- I didn’t see the icons to change the page order. My bad !!
- It might be useful to add an explanation to the Preferences that this setting is only for new books and that for existing books you need to change via the icon bar ?
- Perhaps expand the VIEW menu also so that you can also can change in the view menu the page order from there as well (for people like me who overlook the icons) ?

Thank you for your quick response !

Regards,

Alex

**Comment by mtgto** [comment]
Thanks for the additional information.

Your information revealed a bug where default page order settings are not referenced when opening a book file from the library.
When opening a file directly in the app, the settings are referenced, but when opening a file that has already been added to the app, the settings were not referenced.

This bug has been fixed at https://github.com/mtgto/Fomalhaut2/commit/5b6068fdc63310f3d9571ac1db3312ab0b8ca27d and I submitted to Apple as v1.6.1 https://github.com/mtgto/Fomalhaut2/releases/tag/1.6.1.
